app/models/organisation.rb

```diff
@@ -96,4 +98,13 @@ def display_organisation_attributes
   def has_managing_agents?
     managing_agents.count.positive?
   end
+
+private
+
+  def persist_missing_old_visible_id
+    if old_visible_id.blank?
+      self.old_visible_id ||= "ORG#{id}"
+      update_column(:old_visible_id, old_visible_id)
```

Comment:
On reflection I think it's nice to have one field to distinguish whether an org was migrated or not. Did you consider doing this on the lookup? For example, if `ORG` is appended look at `id`, otherwise `old_visible_id`.

Reply:
```ruby
  scope :filter_by_id, ->(id) { id.start_with?("ORG") ? where(id: id[3..]) : where(old_visible_id: id) }
```

Not as pretty, but something like this?

Reply:
I've done something along those lines and is present in https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/1311